app/main.py

```diff
@@ -16,4 +16,10 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+immutable = [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+mutable = [my_favourite_films, marks, collection_of_coins]
```

Comment:
The list `mutable` correctly contains mutable data structures like lists, dictionaries, and sets. Ensure that this categorization aligns with your intended use of these variables.